examples/use_your_own_data/acm_glem_nc_pretrain.yaml

```diff
@@ -0,0 +1,57 @@
+---
+version: 1.0
+lm_model:
+  node_lm_models:
+    -
+      lm_type: bert
+      model_name: "bert-base-uncased"
+      gradient_checkpoint: false
+      node_types:
+        - paper
+        - author
+        - subject
+gsf:
+  basic:
+    backend: gloo
+    verbose: false
+  lmgnn:
+    lm_train_nodes: 256
+    lm_infer_batch_size: 256
+    freeze_lm_encoder_epochs: 0
+    model_encoder_type: rgcn
+    fanout: "50,50"
+    num_layers: 2
+    hidden_size: 256
+    use_mini_batch_infer: false
+    training_method:
+      name: glem
+      kwargs:
+        em_order_gnn_first: false
+        inference_using_gnn: true
+        pl_weight: 0.5
```

Comment:
Have you tried a lower value (e.g. 0.1)?

Reply:
Yes, I tried 0.1. The performance was not improved, or even worse in some settings.